utilities/transactions/transaction_db_impl.h

```diff
@@ -122,5 +123,39 @@ class TransactionDBImpl : public TransactionDB {
   std::unordered_map<TransactionName, Transaction*> transactions_;
 };
 
+class WriteCommittedTxnDBImpl : public PessimisticTxnDB {
```

Comment:
I am not sure why do we say `WriteCommittedTxnDBImpl` instead of `WriteCommittedTxnDB`
I guess we should not if there is not a `WriteCommittedTxnDB`